src/background/listener/bee-api.listener.ts

```diff
@@ -0,0 +1,85 @@
+import { StoreObserver, getItem } from '../../utils/storage'
```

Comment:
AFAIK until now we always used the camelcase naming conventions and not `-` character. So this should probably be `beeApi.listener.ts`

Reply:
Ah and I actually mystified you. We are not consistent in `bee-js` either. I'll put it on the roundtable.